### PR TITLE
fix:replace beta w/ preview NR-35279

### PR DESCRIPTION
**Pull request by ghost**
Replaced "beta" with "preview".
Jira ticket: https://issues.newrelic.com/browse/NR-35279

### CODE REVIEW

File: src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx [hunk 1]
```diff
@@ -134,7 +134,7 @@ When you edit the site, include the Jira issue key (DOC-1234, for example) in yo
 
       <td>
         * Publication date or due date
-        * Dates for other key events (betas, limited releases, etc.)
+        * Dates for other key events (Limited Preview, Public Preview, General Availability)
```
Comment (bradleycamacho) [inline src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx:137]: Lowercase these

File: src/content/docs/data-apis/ingest-apis/gateway-api-import-data-other-observability-platforms.mdx [hunk 2]
```diff
@@ -7,7 +7,7 @@ redirects:
 Our Gateway API allows users of third-party observability services, like Datadog, to send that data to New Relic.
 
 <Callout variant="important">
-  This feature is currently in **beta** and at this time only supports DataDog. For questions, contact [tsakashita@newrelic.com](mailto:tsakashita@newrelic.com).
+  This feature is currently in **Preview** and at this time only supports DataDog. For questions, contact [tsakashita@newrelic.com](mailto:tsakashita@newrelic.com).
```
Comment (bradleycamacho) [inline src/content/docs/data-apis/ingest-apis/gateway-api-import-data-other-observability-platforms.mdx:10]: Lowercase

File: src/content/docs/nerdgraph-anomaly-detector-configurations-api-tutorial.mdx [hunk 3]
```diff
@@ -8,8 +8,8 @@ metaDescription: How to use New Relic NerdGraph API to query and update anomaly
 
 Automated anomaly detection at New Relic starts with an anomaly detector: a way to systematically observe and record anomalies in a specific dataset. To name and configure the dataset, you'll need to create an anomaly detector configuration, which is a set of fields that tells us the signals you're interested in watching and where to store anomalous data.
 
-<Callout title="BETA FEATURE">
-  This feature is still in development, but we encourage you to try it out!
+<Callout title="PREVIEW FEATURE">
+  This feature is still in Preview, but we encourage you to try it out!
```
Comment (bradleycamacho) [inline src/content/docs/nerdgraph-anomaly-detector-configurations-api-tutorial.mdx:12]: Lowercase

File: src/content/docs/style-guide/word-choice/usage-dictionary.mdx [hunk 4]
```diff
@@ -68,16 +68,16 @@ For consistency across New Relic content, here are our general word usage guidel
...
+    id="Preview"
+    title="Preview"
   >
-    Use the [beta callout](/docs/style-guide/quick-reference/callouts/). In the body text, use lowercase. For example:
+    Use the [Preview callout](/docs/style-guide/quick-reference/callouts/). In the body text, use lowercase. For example:
 
-    <Callout title="BETA FEATURE">
-      This feature is currently in beta.
+    <Callout title="PREVIEW FEATURE">
+      This feature is currently in Preview.
```
Comment (bradleycamacho) [inline src/content/docs/style-guide/word-choice/usage-dictionary.mdx:77]: lowercase

File: src/content/docs/style-guide/writing-docs/processes-procedures/create-edit-content.mdx [hunk 5]
```diff
@@ -67,7 +67,7 @@ If you are comfortable with deleting the page yourself, go for it. Note that if
 
 ## Private edits [#private-edits]
 
-If you need to stage content for a private beta or limited release, contact the docs hero in the #help-documentation Slack channel.
+If you need to stage content for a Preview, contact the docs hero in the #help-documentation Slack channel.
```
Comment (bradleycamacho) [inline src/content/docs/style-guide/writing-docs/processes-procedures/create-edit-content.mdx:70]: lowercase

**Review by bradleycamacho — DISMISSED** [review]
Looks good, just a few small copy edits! I would stay consistent with lowercasing or capitalizing `Preview`.

**Review by ghost — COMMENTED** [review]
Approving this review to lowercase Preview to preview.

File: src/content/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/query-infrastructure-dimensional-metrics-nrql.mdx [hunk 6]
```diff
@@ -9,7 +9,7 @@ metaDescription: 'In New Relic, you can query dimensional metrics for Infrastruc
 
 import namingConvention from 'images/naming-convention.png'
 
-<Callout title="BETA FEATURE">
+<Callout title="PREVIEW FEATURE">
   This feature is currently in beta.
```
Comment (bradleycamacho) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/query-infrastructure-dimensional-metrics-nrql.mdx:13]: Change "This feature is currently in beta." to "This feature is currently in preview".

**Pull request merged**